app/decorators/user_decorator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module UserDecorator
+  def number_of_friends(friends)
```

Comment:
```suggestion
  def number_of_friends
```

`User` model knows its friends already.